adafruit_rgb_display/rgb.py

```diff
@@ -70,9 +70,9 @@ def _block(self, xpos0, ypos0, xpos1, ypos1, data=None):
         return None
     #pylint: enable-msg=too-many-arguments
 
-    def _encode_pos(self, avalue, bvalue):
+    def _encode_pos(self, x, y): #pylint: disable-msg=invalid-name
```

Comment:
I had the impression that pylint has those names added to exceptions in its configuration.

Reply:
I thought so too: https://github.com/adafruit/cookiecutter-adafruit-circuitpython/blob/master/%7B%7B%20cookiecutter.library_name%20%7D%7D/.pylintrc#L303

Reply:
That change got made after i created my template project from cookiecutter.  I have updated the .pylintrc and remove the redundant  \#pylint lines